UCSD/campus-mobile
# Accessibility: Add description to Notices banner

## Summary
For [Accessibility: Add description to Notices banner#1628](https://app.zenhub.com/workspaces/campus-mobile-replatform-59791ae0251e9315c45be418/issues/ucsd/campus-mobile/1628)

## Changelog

[General] [Add]- Notices banners include a description that can be read by screen readers

## Test Plan
- Turn on the screen reader feature on your mobile device (built-in or installed)
- Open the app, click on the top covid-19 test kit vending machine notice banner
- the screen reader should say something like "image - (read the text on the banner)"
